apps/native-component-list/webpack.config.js

```diff
@@ -22,6 +22,15 @@ const locations = {
   modules: absolutePath('../../node_modules'),
 };
 
+let appManifest;
+try {
+  const PWAManifest = require(absolutePath('./web/manifest.json'));
+  appManifest = PWAManifest || {};
+} catch (error) {
+  const nativeAppManifest = require(absolutePath('./app.json'));
```

Comment:
I'm a little hesitant about including app.json for two reasons:
- it may contain some build-related config (e.g. Google API keys) that shouldn't be included in the web JS bundle
- it can be very large and add to the bundle size

I think a better way to go about this would be something like:
```js
function getAppManifest() {
  const rawAppManifest = require(absolutePath('./app.json'));
  return {
    whitelistedField1: rawAppManifest. whitelistedField1, 
    ...
  }
}
```

This way it's less likely we unintentionally bundle up sensitive info or bloat the bundle size.